src/simple-api/src/api/xxENSONOxx.xxSTACKSxx.API/Authentication/JwtBearerAuthenticationConfigurationExtension.cs

```diff
@@ -1,6 +1,6 @@
 ﻿namespace xxENSONOxx.xxSTACKSxx.API.Authentication;
 
-public class JwtBearerAuthenticationConfiguration
+public class JwtBearerAuthenticationConfigurationExtension
```

Comment:
A class name ending with `Extension`?  Was there a problem with the name before?

Reply:
That's a mistake, will revert